runtime/compiler/optimizer/EscapeAnalysis.cpp

```diff
@@ -1489,7 +1527,10 @@ Candidate *TR_EscapeAnalysis::createCandidateIfValid(TR::Node *node, TR_OpaqueCl
             {
             if (trace())
                {
-               printf("secs Class %s implements Runnable in %s\n", getClassName(classNode), comp()->signature());
+               const char *className = getClassName(classNode->getSecondChild());
+               printf("secs Class %s implements Runnable in %s\n",
```

Comment:
please convert to guarded trace

Reply:
@andrewcraik Andrew, are you asking me to replace the `printf` with `traceMsg`?  If so, there are a few other places with calls to `printf` that I'll take care of as well.

Reply:
yes please - printf is not a nice way to debug anything in the JIT so we should only commit code that logs using the traceMsg mechanism.